recipes/abseil/all/conanfile.py

```diff
@@ -145,6 +145,8 @@ def _load_components_from_cmake_target_file(self, absl_target_file_path):
 
         abs_target_content = load(self, absl_target_file_path)
 
+        abs_target_content = abs_target_content.replace("\r\n", "\n")
```

Comment:
Please a comment or link so it's obvious this is required since the next contributor might not know :)